### ISSUE TITLE
Service Mesh - how is ziti different or the same?

### ISSUE THREAD

**Issue by dovholuknf** [openziti/ziti-doc#74]
lots of times we get asked questions about service mesh and how it's the same/different than ziti. let's produce some kind of wiki, doc, article around what ideas overlap, what ideas are different and how it all pertains to ziti

**Comment by cnu1812** [comment]
Would like to work on this!

**Comment by dovholuknf** [comment]
Sounds great! If you want to produce a markdown explaining the similarities and differences and put a pr up against this repo that'd be great and much appreciated.

This repo uses docfx to build the static site but I'd be happy to setup a page somewhere and get review your content and get it published. You can find a very simple bash script at the root of the project that shows you how to build it locally and view the content: https://github.com/openziti/ziti-doc/blob/main/gendoc.sh . The readme talks about it more too.

I would think we would enumerate some popular service mesh solutions, define a service mesh and compare and contrast the solutions. I'm happy to collaborate with you as well.

Welcome to openziti!

**Comment by cnu1812** [comment]
Wondering to know ziti-doc/docfx_project is the working directory for this issue?

**Comment by dovholuknf** [comment]
Yes - that's the "docfx project". inside there is a docfx.json file that declares the overall structure of the static site to generate. I stubbed out a page for you on this branch: https://github.com/openziti/ziti-doc/pull/new/issue-74-doc-svc-mesh

![image](https://user-images.githubusercontent.com/46322585/158816398-2ac71379-f09b-4c4a-bfa3-2a71a08ed1fe.png)

In there I gave the command I run to test it / view it locally as well.
